lib/hocon/impl/simple_config_list.rb

```diff
@@ -21,6 +24,8 @@ def initialize(origin, value, status = ResolveStatus.from_values(value))
     end
   end
 
+  def_delegators :@value, :[], :include?, :empty?, :size, :index, :rindex, :each, :map
+
```

Comment:
I'm :+1: if it's ruby idiomatic. It seems unnecessary to me to reimplement all of these methods if they're just going to call through to `@value` anyway, especially since I highly doubt any of these methods are going to change on the java side.